inventory/sample.casl.rht-labs.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -0,0 +1,29 @@
+---
+deployment_type: openshift-enterprise
+openshift_deployment_type: openshift-enterprise
+
+openshift_master_default_subdomain: "apps.{{ env_id }}.casl.rht-labs.com"
```

Comment:
Please change 'rht-labs.com' to 'example.com'. 
Also, looking at the content below, this shouldn't have been on github.com in the first place. It will now stick in the history - not sure how big of a problem that is. :-)